src/common/KernelBase.cpp

```diff
@@ -701,12 +747,12 @@ void KernelBase::setCaliperMgrVariantTuning(VariantID vid,
     }
     std::string vstr = getVariantName(vid);
     std::string profile = "spot(output=" + od + vstr + "-" + tstr + ".cali";
```

Comment:
This should use output from the `--output` rajaperf argument, if argument provided? Need to look into this